samples/tls/logging.md

```diff
@@ -72,29 +79,40 @@ information about creating this secret manually.
 
 ## Specifying TLS for Ingress
 
-If you are using ingress and want to enable TLS on traffic into the cluster,
-you must manually create two Kubernetes secrets.
+If you are using ingress and want to enable TLS on communications into the cluster,
+you must manually create the following two Kubernetes secrets:
+
+- `elasticsearch-ingress-tls-secret`
+- `kibana-ingress-tls-secret`
 
-1. After you have obtained the certificates for the secrets, use this command
-to generate the secrets:
+**Note:** The process of generating the TLS certificates for these secrets is outside the scope for this example.
+
+1. After you have obtained the TLS certificates for each secret, use the following command to generate the secrets. You must run the command for each secret name.
 
 ```bash
-kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key <cert-key> --cert <cert-file>
+kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
 ```
 
-By default, the value of `namespace` that is used during the deployment process
-is `logging`. Run the command for for each of these values of `secret-name`:
+    where:
```

Comment:
This 'where:' shows up as a code block in rendered markdown. We probably don't want that.

Reply:
Agreed. 
I wanted "where:" indented because it's part of step #1. I can't find a way to do that in markdown that doesn't insert the codeblock. 
I'll just have to make peace with no indent.